packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js

```diff
@@ -111,6 +111,14 @@ export default class Notebook {
     return this.status?.state === NOTEBOOK_STATUS.FAILED;
   }
 
+  isErrored() {
+    return this.status?.state === NOTEBOOK_STATUS.FAILED;
+  }
+
+  isTerminal() {
```

Comment:
Not really getting the meaning here, `isTerminated` ?

Reply:
It means that the notebook is in a terminal state (several of them), `terminated` would be more of a state.